fiftyone/brain/__init__.py

```diff
@@ -712,103 +712,98 @@ def compute_exact_duplicates(
 
 def compute_leaky_splits(
     samples,
-    brain_key=None,
```

Comment:
Happy to chat more offline about the removal of `brain_key`, but my reasoning is that saving a `LeakySplitsIndex` as currently implemented doesn't persist anything of value. It doesn't store the embeddings directly or do any reusable computation that would compel users to call `compute_leaky_splits()` once and then reuse the same object in the future.